### ISSUE TITLE
split chars output is flattened when vectorizing over list

### ISSUE THREAD

**Issue by dandavison** [nushell/nushell#6954]
### Describe the bug

`split chars` vectorizes over list input, but it does so incorrectly (it flattens the result, but there is no justification for it to do that.)

### How to reproduce

```
〉['aa' 'bb'] | split chars | to nuon
[a, a, b, b]
```

### Expected behavior

I expect the result 

```
[[a, a], [b, b]]
```

### Configuration

| key                | value                                           |
| ------------------ | ----------------------------------------------- |
| version            | 0.70.1                                          |
| branch             | dev                                             |
| commit_hash        | e7c5abf4efa0f5645fc19a3bd777044e8a0f7136        |
| build_os           | macos-x86_64                                    |
| build_target       | x86_64-apple-darwin                             |
| rust_version       | rustc 1.63.0 (4b91a6ea7 2022-08-08)             |
| rust_channel       | 1.63.0-x86_64-apple-darwin                      |
| cargo_version      | cargo 1.63.0 (fd9c4297c 2022-07-01)             |
| pkg_version        | 0.70.1                                          |
| build_time         | 2022-10-20 09:10:05 -04:00                      |
| build_rust_channel | release                                         |
| features           | database, dataframe, default, trash, which, zip |
| installed_plugins  |                                                 |

**Comment by dandavison** [comment]
This applies to `split row` also (is there a difference between `split chars` and `split row ''`?)

**Comment by webbedspace** [comment]
My opinion is that the `split` commands simply shouldn't iterate over list<str>, instead requiring explicit `each` to do so:
```
〉['aa' 'bb'] | each {split chars} | to nuon
[[a, a], [b, b]]
```

**Comment by dandavison** [comment]
Yes, I was surprised when I saw that many nushell commands vectorize automatically over input lists. My initial reaction was that no commands should do that since nushell has `each`, but have you seen how many do it -- they're marked as `vectorizes_over_list(true)` in https://github.com/nushell/nushell/pull/6796/files. So, if we don't get rid of the behavior, it definitely needs to be made very consistent and documented as a prominent part of the language. I'm hoping that having the vectorizing behavior documented explicitly in help text and queryable in command queries will help one way or the other.

**Comment by sholderbach** [comment]
Fixed by #9739
